engines/bladerunner/bladerunner.cpp

```diff
@@ -1164,7 +1164,7 @@ void BladeRunnerEngine::handleKeyUp(Common::Event &event) {
 		return;
 	}
 
-	if (_vqaIsPlaying) {
+	if (_vqaIsPlaying && event.kbd.keycode == Common::KEYCODE_ESCAPE) {
```

Comment:
please, can you add "KEYCODE_RETURN" as well as that is used for different skips

Reply:
I added that and I also added a check to make sure Alt isn't pressed.